### PR TITLE
Fixed empty variable issue in header template

### LINKED ISSUE #8942: Invalid expression in wagtail-4.0 (dev20220804) header template
### Issue Summary

I'm trying out the [wagtail nightly](https://releases.wagtail.org/nightly/dist/wagtail-4.0.dev20220804-py3-none-any.whl) to see if my project works with the progress so far on wagtail 4

I have this line added to my template settings that will render a string if the there is an invalid expression in the template `"string_if_invalid": "INVALID EXPRESSION: [%s]"` (recommended in [two-scoops-of-django-3-x](https://www.feldroy.com/books/two-scoops-of-django-3-x)).

Since updating to the nightly it renders `INVALID EXPRESSION: [extra_actions]` in several page and model headers as can be seen in the screenshot below.

Screen Shot 2022-08-04 at 12 18 07

Is there an issue in the new header template?
Perhaps just needs to check if extra_actions exists like `{% if extra_actions %}{{ extra_actions }}{% endif %}`

### Steps to Reproduce

1. add `"string_if_invalid": "INVALID EXPRESSION: [%s]"` to template settings:
```
TEMPLATES = [
    {
        "BACKEND": "django.template.backends.django.DjangoTemplates",
        "OPTIONS": {
            "context_processors": [...],
            "string_if_invalid": "INVALID EXPRESSION: [%s]",
        },
    },
]
```
2. Navigate to admin

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

### Technical details

-   Python version: 3.10.6
-   Django version: 3.2.15
-   Wagtail version: wagtail-4.0.dev20220804-py3-none-any.whl
-   Browser version: Firefox 103

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixes #8942.

### CODE REVIEW

File: wagtail/admin/templates/wagtailadmin/shared/header.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1 class="w-header__title" id="header-title">
                     {% elif avatar %}
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
-                    {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {{ title }}{% if subtitle %}<span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
```
Comment (zerolab) [inline wagtail/admin/templates/wagtailadmin/shared/header.html:32]: this was correct. you do want a space between the title and the subtitle

Comment (PaarthAgarwal) [reply wagtail/admin/templates/wagtailadmin/shared/header.html:32]: 🤔
Yes

Comment (PaarthAgarwal) [reply wagtail/admin/templates/wagtailadmin/shared/header.html:32]: Indeed yes. These spacing things with Django tags always confuse me. Thanks @zerolab!

**Review by zerolab — APPROVED** [review]

**Comment by lb-** [comment]
Thanks @PaarthAgarwal  - appreciate the quick investigation and follow up on this.

**Pull request merged**